com.unity.render-pipelines.high-definition/Documentation~/deep-learning-super-sampling-in-hdrp.md

```diff
@@ -10,52 +10,59 @@ This section includes HDRP-specific requirements and compatibility information f
 
 HDRP supports DLSS on the following platforms:
 
-DirectX 11 on Windows 64 bit
-DirectX 12 on Windows 64 bit
-Vulkan on Windows 64 bit
-HDRP does not support DLSS for Metal, Linux, Windows using x86 architecture (Win32), or any other platform.
+* DirectX 11 on Windows 64 bit
+* DirectX 12 on Windows 64 bit
+* Vulkan on Windows 64 bit
+
+**Note**: HDRP doesn't support DLSS for Metal, Linux, Windows using x86 architecture (Win32), or any other platform.
 
 To build your project for Windows, use x86_64 architecture (Win64).
 
 For information about the hardware requirements of DLSS, see [NVIDIA'S DLSS requirements](https://developer.nvidia.com/nvidia-dlss-access-program).
 
 ## Using DLSS
 
-To use DLSS in your scene, you must:
+To use DLSS in your scene:
 
-1. Add the NVIDIA package.
-2. Enable DLSS in your HDRP Asset.
-3. Enable DLSS for each Camera you want to use it with.
-4. Set the DLSS quality mode.
+1. Add the NVIDIA package. You can either do this automatically or manually.
+
+    * To install the NVIDIA package automatically:
 
-### Adding the NVIDIA package
+        1. Select an [HDRP Asset](HDRP-Asset.md) and view it in the Inspector.
+        2. Go to **Rendering** > **Dynamic Resolution** and click **Install NVIDIA Package**.
 
-To add the NVIDIA package to your Unity project, there are two methods you can use. To install it automatically:
+    * To install the NVIDIA package manually:
 
-1. Select an [HDRP Asset](HDRP-Asset.md) and view it in the Inspector.
-2. Go to **Rendering** > **Dynamic Resolution** and click **Install NVIDIA Package**.
+        1. Open the [Package Manager window](https://docs.unity3d.com/Manual/upm-ui.html) (menu: **Window** > **Package Manager**).
+        2. Select **Packages**, then select **Built-in**.
+        3. In the packages list view, find and select the NVIDIA package.
+        4. In the bottom right of the package-specific detail view, select **Enable**.
 
-To install it manually:
+2. Enable DLSS in your HDRP Asset.
+
+    1. Select the HDRP Asset you want to enable DLSS for and view it in the Inspector.
+    2. Go to **Rendering** > **Dynamic Resolution** and select **Enable**.
+    3. In the dynamic resolution section, select the **Enable DLSS** property to expose other properties that you can use to customize DLSS. For information about these properties, see the [HDRP Asset](HDRP-Asset.md) documentation.
```

Comment:
The dynamic resolution title should be capitalized in the same way it is in the UI and bolded.